src/neo-vm/OpCode.cs

```diff
@@ -233,14 +233,34 @@ public enum OpCode : byte
         /// Pop the top value of the stack, if it false, then exit vm execution and set vm state to FAULT.
         /// </summary>
         ASSERT = 0x38,
+        /// <summary>
+        /// Pop the top value of the stack, and throw it.
+        /// </summary>
         THROW = 0x3A,
-        //TRY = 0x3B,
-        //TRY_L = 0x3C,
-        //ENDT = 0x3D,
-        //ENDC = 0x3E,
-        //ENDF = 0x3F,
         /// <summary>
-        /// Returns from the current method.
+        /// TRY CatchOffset(sbyte) FinallyOffset(sbyte). If there's no catch body, set CatchOffset 0. If there's no finally body, set FinallyOffset 0.
+        /// </summary>
+        [OperandSize(Size = 2)]
+        TRY = 0x3B,
+        /// <summary>
+        /// TRY_L CatchOffset(int) FinallyOffset(int). If there's no catch body, set CatchOffset 0. If there's no finally body, set FinallyOffset 0.
```

Comment:
If there are multiple catches? or it should not be allowed more than one?

Reply:
I think it's enough at present, we don't have exception type.

Reply:
Multiple catches can be supported by the compiler. It can use `if...else` or `ISTYPE` in the `catch` section.